images/ansible-operator/pipfile.Dockerfile

```diff
@@ -15,12 +15,20 @@ ENV PIP_NO_CACHE_DIR=1 \
     PIPENV_CLEAR=1
 # Ensure fresh metadata rather than cached metadata, install system and pip python deps,
 # and remove those not needed at runtime.
-RUN set -e && yum clean all && rm -rf /var/cache/yum/* \
-  && yum update -y \
-  && yum install -y libffi-devel openssl-devel python39-devel gcc python39-pip python39-setuptools \
+RUN set -e && dnf clean all && rm -rf /var/cache/dnf/* \
+  && dnf update -y \
+  && dnf install -y gcc libffi-devel openssl-devel python3.12-devel \
+  # The python3 binary is bound to python3.9, which is preinstalled in the image.
+  # Bind python3 to python3.12 at a location that comes sooner in the PATH variable.
+  # This is carried over in the final image when this directory path is copied.
+  && pushd /usr/local/bin && ln -sf ../../bin/python3.12 python3 && popd \
+  # Confirm python3.12 here.
+  && python3 --version \
```

Comment:
Should this also be in the other docker file also? As well as the comments before the `pushd`?

Reply:
I needed to sync the two files. Technically, they don't match exactly (even for the aligned phase), but the parts that do match were mismatched. Now it should look more like it did before, but with the relevant bits (e.g. image ref, installed package workflow) updated.